operator/v1/types_dns.go

```diff
@@ -30,6 +30,47 @@ type DNS struct {
 
 // DNSSpec is the specification of the desired behavior of the DNS.
 type DNSSpec struct {
+	// servers is a list of DNS resolvers that provide name query delegation for one or
+	// more subdomains outside the scope of the cluster domain. If servers consists of
+	// more than one Server, longest suffix match will be used to determine the Server.
+	//
+	// For example, if there are two Servers, one for "foo.com" and another for "a.foo.com",
+	// and the name query is for "www.a.foo.com", it will be routed to the Server with Zone
+	// "a.foo.com".
+	//
+	// If this field is nil, no servers are created.
```

Comment:
Will the operator actually create servers or is the server list simply a reference to a server that already exists?